client/src/actions/friends.js

```diff
@@ -121,3 +122,21 @@ export const getIncomingFriendRequests = () => async dispatch => {
 export const checkFriendshipStatus = targetUsername => {
   return apiRequest.get('/users/friends/status/' + targetUsername)
 }
+
+export const loadActiveFriends = (page, size) => async dispatch => {
+  dispatch({
+    type: FRIENDS_STARTED_LOADING
```

Comment:
Are you sure about this event? This dispatch will trigger all preloaders for friends (e.g. 3x3) in other windows on page, not only in a bar on the right side